ksql-execution/src/main/java/io/confluent/ksql/execution/ddl/commands/KsqlTopic.java

```diff
@@ -17,23 +17,23 @@
 
 import static java.util.Objects.requireNonNull;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.errorprone.annotations.Immutable;
 import io.confluent.ksql.serde.KeyFormat;
 import io.confluent.ksql.serde.ValueFormat;
 
 @Immutable
 public class KsqlTopic {
-
   private final String kafkaTopicName;
   private final KeyFormat keyFormat;
   private final ValueFormat valueFormat;
   private final boolean isKsqlSink;
 
   public KsqlTopic(
-      final String kafkaTopicName,
-      final KeyFormat keyFormat,
-      final ValueFormat valueFormat,
-      final boolean isKsqlSink
+      @JsonProperty(value = "kafkaTopicName", required = true) final String kafkaTopicName,
```

Comment:
I don't think we should be exposing `KsqlTopic`: it's a implementation detail.  (Tis OK to address this in follow up PR).